test/plugin/scenarios/jedis-2.x-scenario/support-version.list

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# 3.0.1 not support stream
+3.0.1
```

Comment:
What does this mean?

Reply:
And is 3.0.1 the only 3.x version?

Reply:
Stream supported since Jedis 3.1.0 , So I put `3.0.1` to `  jedis-2.x-scenario` , This  test-scenario not  include  stream  test  
https://github.com/redis/jedis/releases/tag/jedis-3.1.0

Reply:
I see. Thanks for sharing.